src/js/view/controls/components/menu/menu.js

```diff
@@ -123,6 +124,12 @@ export default class Menu extends Events {
         prependChild(this.mainMenu.topbar.el, backButton.element());
         return backButton;
     }
+    setBackButtonAriaLabel(labelText) {
+        const backButtonElement = this.mainMenu.backButton.element();
+        const localizedPrevious = this.localization.prev;
+        const backButtonText = labelText ? localizedPrevious + ' - ' + labelText : localizedPrevious;
```

Comment:
One question here -  the `labelText` I add is _not_ localized - but I'm not sure there's anything I can do about that, unless we also updated the corresponding localization options.

Reply:
menu.title is already localized, should be good :)